packages/wdio-allure-reporter/src/utils.js

```diff
@@ -1,7 +1,8 @@
 import process from 'process'
 import CompoundError from './compoundError'
 import { testStatuses, mochaEachHooks, mochaAllHooks, linkPlaceholder } from './constants'
-
+// eslint-disable-next-line no-control-regex
+const regex = new RegExp(/[\u001b\u009b][[()#;?]*(?:[0-9]{1,4}(?:;[0-9]{0,4})*)?[0-9A-ORZcf-nqry=><]/g)
```

Comment:
can we comment where this regex comes from or is it build by you?

Reply:
I used ansi-strip instead of regex in new commit.